### ISSUE TITLE
Imgur becomes dark after a while

### ISSUE THREAD

**Issue by MSReporter** [Metastruct/issues#348]
example: https://i.imgur.com/QYvV79o.png

Bug reported by 'twentysix' (STEAM_0:0:70542806).

**Comment by Etothepowerof26** [comment]
19:16 - Atlas: twentysix it also happens with SCAP

**Comment by bjoern90** [comment]
and it happes after the core meltdown like the explosion screen effect

**Comment by ghost** [comment]
Update on this (apparently people knew about this but no-one updated the issue):
From testing with another player who reported that the imgur screen went dark after using items such as vodka, I managed to narrow it down that the issue is being caused by RenderScreenspaceEffects hooks, particularly any of those using DrawMotionBlur function in the hook (hence things like the mgn explosion effect and what not causing this). A temporary fix I found with another player to get the screen clear again is just to DrawMotionBlur with values of 1,1,0 in a RenderScreenspaceEffects hook and then remove the hook.

TLDR: Still dont know why this is happening fully so adding help-wanted label, but at least there is a temp fix for players who want it.

**Comment by ghost** [comment]
https://i.imgur.com/vBLboGy.png

**Comment by Zeni44** [comment]
Did some more research into this.

```
] lua_run print(Material("pp/fb"):GetFloat("$alpha"))
0.0099999997764826
```


Turns out that `DrawMotionBlur` does not properly reset the alpha value of the material and therefore results in the material becoming alpha-less (therefore making it render black).
Manually setting `$alpha` back to 1 fixes it.

**Comment by Zeni44** [comment]
Fixed.
